app/styles/_ember-osf.scss

```diff
@@ -0,0 +1,21 @@
+// Internal Styles
```

Comment:
So this and `addon.scss` both need to be updated whenever a new component is added? That's kinda annoying, but I guess only slightly more than before.